curvefs/conf/client.conf

```diff
@@ -114,6 +114,19 @@ fuseClient.maxDataSize=1024
 fuseClient.refreshDataIntervalSec=30
 fuseClient.warmupThreadsNum=10
 
+# the write throttle bps of fuseClient, default no limit
+fuseClient.throttle.avgWriteBytes=0
+# the write burst bps of fuseClient, default no limit
+fuseClient.throttle.burstWriteBytes=0
+# the times that write burst bps can continue, default 180s
+fuseClient.throttle.burstSecs=180
+# the write throttle iops of fuseClient, default no limit
+fuseClient.throttle.avgWriteIops=0
+# the read throttle bps of fuseClient, default no limit
+fuseClient.throttle.avgReadBytes=0
+# the read throttle iops of fuseClient, default no limit
+fuseClient.throttle.avgReadIops=0
```

Comment:
burst should contains four types:
  - read
      - bps
      - iops
  - write
      - bps : just this one here, we'd better add anothers.
      - iops

Reply:
I means we can fix this by a new pr.